app/navigation/Navigator.scala

```diff
@@ -251,6 +251,20 @@ class Navigator @Inject()() {
     case _ => getCheckYourAnswers
   }
 
+  private val repayNormalRouteMap: Page => UserAnswers => Call = {
```

Comment:
can you change this to a def that takes mode as a parameter instead of having 2 variable declaration?

Reply:
How can I pass only mode, we need to have a page as well. I have just followed the existing pattern.

Reply:
private def routeMap(mode: Mode): Page => UserAnswers => Call = {

Reply:
Thanks, fixed it.